Integrations/URLHaus/URLHaus.py

```diff
@@ -0,0 +1,493 @@
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+
+''' IMPORTS '''
+import os
+import traceback
+import requests
+import zipfile
+import io
+from datetime import datetime as dt
+
+# Disable insecure warnings
+requests.packages.urllib3.disable_warnings()
+
+''' GLOBALS/PARAMS '''
+
+# Remove trailing slash to prevent wrong URL path to service
+API_URL = demisto.params()['url'].rstrip('/')
+
+# Should we use SSL
+USE_SSL = not demisto.params().get('insecure', False)
+
+# Remove proxy if not set to true in params
+if not demisto.params().get('proxy'):
+    os.environ.pop('HTTP_PROXY', None)
+    os.environ.pop('HTTPS_PROXY', None)
+    os.environ.pop('http_proxy', None)
+    os.environ.pop('https_proxy', None)
+
+THRESHOLD = int(demisto.params().get('threshold', 1))
+
+COMPROMISED_IS_MALICIOUS = demisto.params().get('compromised_is_malicious', False)
+
+# Headers to be sent in requests
+HEADERS = {
+    'Content-Type': 'application/x-www-form-urlencoded',
+    'Accept': 'application/json'
+}
+
+''' HELPER FUNCTIONS '''
+
+
+def http_request(method, command, data=None):
+    url = f'{API_URL}/{command}/'
+    demisto.info(f'{method} {url}')
```

Comment:
We don't need this log here, maybe in debug and if so then add something more indicative